paddle/fluid/operators/conv_cudnn_op.cu.cc

```diff
@@ -158,6 +158,12 @@ class CUDNNConvOpKernel : public framework::OpKernel<T> {
     PADDLE_ENFORCE_LE(workspace_size_in_bytes, workspace_size_limit,
                       "workspace_size to be allocated exceeds the limit");
 
+    void* cudnn_workspace = nullptr;
+    auto place = boost::get<platform::CUDAPlace>(ctx.GetPlace());
+    if (framework::GetEagerDeletionThreshold() >= 0) {
```

Comment:
It seems that these lines of code is not related with the main logic of `Eager Deletion`